src/web/index.js

```diff
@@ -290,7 +295,7 @@ if (!params.theme) {
           // Skip updating history for this theme, because it came from the URL
           skipHistory: true,
           // Skip updating the add-on for this theme, because it needs approval
-          skipAddon: true
+          skipAddon: false
```

Comment:
Why was this change made?

This line of code contradicts the code comment above. With this proposed change, opening a shared theme will unconditionally override the user's current theme without prompting. That's undesired.

When a shared theme is opened, there should be some UI from where the user can confirm that they want to apply the new theme. This is implemented in the "pending theme" feature in Firefox Color. If the prompt does not appear as expected, I suggest to look into its implementation to try and figure out why.